IdentityCore/src/cache/accessor/MSIDDefaultTokenCacheAccessor.m

```diff
@@ -918,4 +918,32 @@ - (MSIDAppMetadataCacheItem *)getAppAppMetadataForConfiguration:(MSIDConfigurati
     return [_accountCredentialCache getAppMetadata:metadataKey context:context error:error];
 }
 
+- (BOOL)removeAppMetadata:(MSIDAppMetadataCacheItem *)appMetadata
```

Comment:
I assume this will only be used for the test app? Or why else we would ever delete it?

Reply:
Yes this is only used for test app. We will never need to delete app metadata, just create it once and update in cases of client_mismatch.